src/js/components/DataSearch/propTypes.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    property: PropTypes.string,
```

Comment:
I might be missing something but I didn't see `property` in DataSearch.js so why do we need this?

Reply:
fixed